src/Handler/ExceptionHandler.php

```diff
@@ -104,6 +104,8 @@ public function __invoke(
             $handler = $this->handler($type);
             $this->whoops->pushHandler($handler);
 
+            $this->whoops->writeToOutput(false);
+            $this->whoops->allowQuit(false);
```

Comment:
@shadowhand (feelsbadman) you're 100% right, somehow my application was missing the whoops configuration. Nothing to see here...